plugins/steelseries/steelseries.quirk

```diff
@@ -12,6 +12,12 @@ GType = FuSteelseriesSonic
 Icon = input-mouse
 
 # Aerox 3 Wireless
+[HIDRAW\VEN_0111]
```

Comment:
this seems over-broad; I think you want a DEV_ there, no? At the moment it's matching all HID devices with that vendor.

Reply:
Sure, this is a draft; I have voluntary dropped for the testing.